### PR TITLE
settings page

### PR DESCRIPTION
**Pull request by jwang711**

### CODE REVIEW

**Review by jeffacuario — APPROVED** [review]
great job

File: website/templates/settings/settings.html [hunk 1]
```diff
@@ -1,5 +1,114 @@
...
-{% block title %}Settings{% endblock %} 
-{% block content %}
-<h1>Settings</h1>
-{% endblock %}
\ No newline at end of file
+{% extends "base.html" %}
+{% block title %}Settings{% endblock %}
+<!doctype html>
+<html>
+<body>
```
Comment (thesteau) [inline website/templates/settings/settings.html:5]: Looks good, however, the html, bod, and the !docttype are inherited from the base, so those lines can be removed.

Comment (thesteau) [inline website/templates/settings/settings.html:5]: Looks good, however, the html, bod, and the !docttype are inherited from the base, so those lines can be removed.

File: website/templates/settings/settings.html [hunk 2]
```diff
@@ -1,5 +1,114 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    {% endblock %}
+</body>
+
+</html>
```
Comment (thesteau) [inline website/templates/settings/settings.html:114]: Likewise to the /body, /html tag - these are inherited from base

Comment (thesteau) [inline website/templates/settings/settings.html:114]: Likewise to the /body, /html tag - these are inherited from base

File: website/templates/settings/settings.html (hunk 1 above)
Comment (jeffacuario) [reply website/templates/settings/settings.html:5]: do we need to revert?

Comment (thesteau) [reply website/templates/settings/settings.html:5]: No, it's fine. It can be changed later.

image

Comment (jwang711) [reply website/templates/settings/settings.html:5]: Thanks guys. I have made the changes on my end.

**Pull request merged**